wordpress-mobile/WordPress-iOS
# Introducing PingHub (without friends, but for 6.8)

I'm giving up on the approach in #6331, for now, and adding the files to the main project/target, and managing dependencies with Cocoapods. I copy from there:

This is a first basic approach to #6164. I've added the PingHub client and very basic logic:
- Connect when the app launches/log in/foreground
- Disconnect when app goes to background/sign out
- On push, sync the note pushed
- On delete, delete the note deleted

Things I haven't done yet:

- Reconnect when connection drops, reachability
- Coalescing multiple push events into a full sync
- Smarter sync so we don't always sync when notifications appear as long as PingHub is connected

It will probably take care of #6290 and other similar issues but we'll have to verify.

To test:

- Launch the app, switch to notifications tab
- From a browser with another account, like a post, the like notification should appear immediately.
- From the browser, unlike the post, the like notification should disappear immediately.

Needs review: @jleandroperez
